backend/routes/PostReaction/AttendeeHelperFunction.py

```
@@ -0,0 +1,59 @@
+    rsvp = get_user_rsvp(db, event_id, user_id)
+    if not rsvp:
+        raise HTTPException(status_code=404, detail="You haven't RSVP'd for this event.")
+    
+    db.delete(rsvp)
+    db.commit()
+
+def count_rsvp_status(db: Session, event_id: int, status: str) -> int:
+    """Count users for a given RSVP status (going/interested)."""
+    return db.query(func.count()).filter(
+        EventAttendee.event_id == event_id,
+        EventAttendee.status == status
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)